exampleSite/content/components/adopters/ecdtools.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "ECD Tools Project Adopters"
+date: 2019-09-10T15:50:25-04:00
+working_group: cloud-development-tools
+hide_sidebar: true
+---
+
+{{< eclipsefdn_adopters >}}
```

Comment:
I would expect you to set the working group with the shortcode not in the page param.

{< eclipsefdn_adopters working_group="cloud-development-tools" >}}

Reply:
updated